spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestPathIdentifier.java

```diff
@@ -75,12 +74,11 @@ public void testPathIdentifier() throws TableAlreadyExistsException, NoSuchTable
             sparkCatalog.createTable(
                 identifier, SparkSchemaUtil.convert(SCHEMA), new Transform[0], ImmutableMap.of());
 
-    Assert.assertEquals(table.table().location(), tableLocation.getAbsolutePath());
-    Assertions.assertThat(table.table()).isInstanceOf(BaseTable.class);
-    Assertions.assertThat(((BaseTable) table.table()).operations())
-        .isInstanceOf(HadoopTableOperations.class);
+    assertThat(table.table().location()).isEqualTo(tableLocation.getAbsolutePath());
```

Comment:
actual/expected is the wrong way here and in `assertThat(sparkCatalog.loadTable(identifier)).isEqualTo(table);`